homeassistant/components/switch/__init__.py

```diff
@@ -105,6 +109,16 @@ def today_power_mw(self):
         return None
 
     @property
+    def standby_state(self):
+        """ Is the device on - or in standby. """
+        return None
+
+    @property
+    def sensor_state(self):
+        """ Is the sensor on or off. """
```

Comment:
I remember you telling me this before, sorry for forgetting.  Let's keep the sensor state in it for now as Home Assistant right now does not have a good solution yet for devices that can be represented in multiple categories.

You see, in a perfect world you would want the sensor to be visible as a sensor so that you can see graphs etc.

I guess you could have Wemo entity also spawn up a normal entity that represents the sensor state, which updates when the parent updates. Or just leave it for now and we'll tackle that problem another day.